### ISSUE TITLE
[metricbeats] Enrich statistics about swap and memory

### ISSUE THREAD

**Issue by unixsurfer** [elastic/beats#13744]
**Describe the enhancement:**
Add the following metrics to the list of exposed memory/swap metrics
`pgscank` = `pgscan_kswapd` from `/proc/vmstat`, number of pages scanned by the kswapd daemon
`pgscand` = `pgscan_direct` from `/proc/vmstat`, number of pages scanned directly
`pgfreed` = `pgfree` from `/proc/vmstat`,  number of pages freed by the system
`pgsteal` = `pgsteal` from `/proc/vmstat`,  Number of pages placed on the free list by the system
We call [GetVMStat](https://github.com/elastic/beats/blob/master/metricbeat/module/system/memory/memory.go#L78) to fetch a ` VMStatInfo` struct but we only fetch few metrics, while [GetVMStat](https://github.com/elastic/beats/blob/master/libbeat/metric/system/memory/memory.go#L157) returns the aforementioned [struct](https://github.com/elastic/go-sysinfo/blob/master/types/host.go#L96) which has all metrics from `/proc/vmstat.`

Moreover, it would be great if you can caluclate the `vmeff` metric that `sar` tool list:
```
  %vmeff
                     Calculated as pgsteal / pgscan, this is a metric of the efficiency  of  page
                     reclaim.  If  it  is near 100% then almost every page coming off the tail of
                     the inactive list is being reaped. If it gets too low (e.g. less  than  30%)
                     then  the virtual memory is having some difficulty.  This field is displayed
                     as zero if no pages have been scanned during the interval of time.
```
**Describe a specific use case for the enhancement or feature:**
These metrics will help us to have visibility in memory saturation cases

**Comment by unixsurfer** [comment]
Ideally, it would be great to expose everything that `VMStatInfo` struct contains.

**Comment by fearful-symmetry** [comment]
@unixsurfer these changes have been merged and should be in 7.6. I'm gonna close this, if there's any related changes can we open it in another issue?

**Comment by unixsurfer** [comment]
@fearful-symmetry Thanks a lot for work on this. It is very much appreciated.
